### PR TITLE
Work section

### PR DESCRIPTION
**Pull request by Aliahmad-karimi**
add work section
add about me section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,7 +40,121 @@ <h1>I'm Ali Ahmad Karimi Glad to see you!</h1>
...
+                <p class="ali1">Ali Ahmad</p><p class="det"> &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2020</p>
+            </div>
+            <div class="parg">
+                <p>An open source code for every coding language. W3 school is one the most usefull websites.</p>
+            </div>
+            <div class="lan">
+                <p class="lang">html</p>
+                <p class="lang">css</p>
+                <p class="lang">javascript</p>
+            </div>
```
Comment (mukhammadrizooff) [inline index.html]: - [x] [REQUIRED] Please make changes! The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li`.

File: index.html [hunk 2]
```diff
@@ -40,7 +40,121 @@ <h1>I'm Ali Ahmad Karimi Glad to see you!</h1>
...
+                <p class="ali1">Ali Ahmad</p><p class="det"> &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2020</p>
+            </div>
+            <div class="parg">
+                <p>This is a sample from Youtube. It is copy and just for the purpose of demo project.</p>
+            </div>
+            <div class="lan">
+                <p class="lang">html</p>
+                <p class="lang">css</p>
+                <p class="lang">javascript</p>
+            </div>
```
Comment (mukhammadrizooff) [inline index.html]: - [x] [REQUIRED] Please make changes! The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li`.

File: index.html [hunk 3]
```diff
@@ -40,7 +40,121 @@ <h1>I'm Ali Ahmad Karimi Glad to see you!</h1>
...
+                <p class="ali1">Ali Ahmad</p><p class="det"> &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2020</p>
+            </div>
+            <div class="parg">
+                <p>This is a sample from A website that teach us how to learn Grid in a game manner.</p>
+            </div>
+            <div class="lan">
+                <p class="lang">html</p>
+                <p class="lang">css</p>
+                <p class="lang">javascript</p>
+            </div>
```
Comment (mukhammadrizooff) [inline index.html]: - [x] [REQUIRED] Please make changes! The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li`.

File: index.html [hunk 4]
```diff
@@ -40,7 +40,121 @@ <h1>I'm Ali Ahmad Karimi Glad to see you!</h1>
...
+                <p class="ali1">Ali Ahmad</p><p class="det"> &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2020</p>
+            </div>
+            <div class="parg">
+                <p>This is website that teach flexbox in a game manner. They are just sample for my project.</p>
+            </div>
+            <div class="lan">
+                <p class="lang">html</p>
+                <p class="lang">css</p>
+                <p class="lang">javascript</p>
+            </div>
```
Comment (mukhammadrizooff) [inline index.html]: - [x] [REQUIRED] Please make changes! The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li`.

File: index.html [hunk 5]
```diff
@@ -40,7 +40,121 @@ <h1>I'm Ali Ahmad Karimi Glad to see you!</h1>
...
+            <div class="social">
+                <p class="connect">LET’S CONNECT</p>
+                <ul class="social-ul">
+                    <li><a href=""><img src="images/Twitter.png" alt="Twitter"></a></li>
+                    <li><a href=""><img src="images/Github.png" alt="Github" class="sIcon"></a></li>
+                    <li><a href=""><img src="images/Linkedin-icon.png" alt="Linkedin" class="sIcon"></a></li>
+                </ul>
+                <div class="resume">
+                    <button type="button" class="resumee">Get my resume</button>
+                </div>
```
Comment (mukhammadrizooff) [inline index.html:140]: - [x] [REQUIRED] The button (e.g., "Get my resume") should follow the button styling: Active, Hover and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1))

image

File: style.css [hunk 6]
```diff
@@ -116,3 +123,260 @@ ul {
...
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  margin-top: 0;
+  margin-bottom: 0;
+}
```
Comment (mukhammadrizooff) [inline style.css]: - [x] [REQUIRED] Please add this styles `
width: 327px;
height: 101px;
font-family: 'Poppins';
font-style: normal;
font-weight: 700;
font-size: 40px;
line-height: 52px;
display: flex;
align-items: center;
color: #172B4D;`! 

**The picture on the left is yours**

image

File: style.css [hunk 7]
```diff
@@ -116,3 +123,260 @@ ul {
...
+  margin-bottom: 12px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  margin-left: 0;
+  margin-top: 12px;
+}
```
Comment (mukhammadrizooff) [inline style.css]: - [ ] [REQUIRED] Good job so far! Please make the changes shown in the images below =>

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Aliahmad-karimi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good UI design ✔️ 
- Descriptive commit history ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @Aliahmad-karimi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [x] No linters errors :heavy_check_mark: 
- [x] Good commit history :heavy_check_mark: 
- [x] Descriptive Pull Resquest :heavy_check_mark: 
- [x] Good implementation of all changes required :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**